src/Symfony/Component/HttpFoundation/HeaderBag.php

```diff
@@ -197,14 +179,11 @@ public function remove($key)
     /**
      * Returns the HTTP header value converted to a date.
      *
-     * @param string    $key     The parameter key
-     * @param \DateTime $default The default value
-     *
      * @return \DateTime|null The parsed DateTime or the default value if the header does not exist
      *
      * @throws \RuntimeException When the HTTP header is not parseable
      */
-    public function getDate($key, \DateTime $default = null)
+    public function getDate(string $key, \DateTime $default = null)
```

Comment:
we should probably change \DateTime to \DateTimeInterface and adjust the return phpdoc as well. but that could be a separate PR.